helix-term/src/commands.rs

```diff
@@ -414,7 +414,8 @@ impl MappableCommand {
         decrement, "Decrement",
         record_macro, "Record macro",
         replay_macro, "Replay macro",
-        command_palette, "Open command palette",
+        command_palette, "Open command pallete",
+        toggle_syntax_highlighting, "Toggle syntax highlighting for current document",
```

Comment:
We  don't need a separate command, it should be the equivalent of setting the language to none/unset via the set language command

Reply:
Hmm though yeah this would disable lsp and other integrations too